code/modules/mob/living/carbon/shock.dm

```diff
@@ -8,8 +8,7 @@
 	1	* src.getToxLoss() + 		\
 	1	* src.getFireLoss() + 		\
 	1	* src.getBruteLoss() + 		\
-	1	* src.getCloneLoss() + 		\
-	1	* src.halloss
+	1	* src.getCloneLoss()
```

Comment:
Only in Bay-code it's been like this since forever, where the single 1 is there mostly for consistency:
```
	src.traumatic_shock = 			\
	1	* src.getOxyLoss() + 		\
	0.7	* src.getToxLoss() + 		\
	1.5	* src.getFireLoss() + 		\
	1.2	* src.getBruteLoss() + 		\
	1.7	* src.getCloneLoss() + 		\
	2	* src.halloss + 			\
	-1	* src.analgesic
```

According to your very own Git history the source of the change to all 1s was apparently made by a certain @Fox-McCloud in the commit 840e1fc5c3c0242ab598c2a02c25b6c2980a0936
#BlameYourself